cli/changelog/0.20.0.md

```diff
@@ -0,0 +1,12 @@
+A release of 0.20.0
+
+### Features
+
+- Adds a `@graphql` directive to support generating queries and mutations from upstream GraphQL APIs.
+- Adds a `JWKS` auth provider.
+- Adds initial support for TypeScript configurations.
+
+### Bug Fixes
+
+- Defaults on OpenAPI enums now work correctly.
+- Significantly reduced memory use of introspection queries.
```

Comment:
These were already released as part of 0.19.1

Reply:
Copied from rc.1 PR. Re-reading all changes to a better list.